test/gcs/entrypoint.init_test.sh

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash -e
 
-# Expand all variables in "/tmp/.boto"
-envsubst < /tmp/.boto > /root/.boto
+if [ -n "${GCP_PROJECT_ID}" ] && [ -n "${GCP_SERVICE_ACCOUNT_KEY_JSON_PATH}" ]; then
+  # Using GCP service account authorization
+  gcloud auth activate-service-account --key-file="${GCP_SERVICE_ACCOUNT_KEY_JSON_PATH}"
+  gcloud --quiet config set project ${GCP_PROJECT_ID}
+elif [ -n "${GCP_ACCESS_KEY_ID}" ] && [ -n "${GCP_SECRET_ACCESS_KEY}" ]; then
+  # Using HMAC authorization
+  # Expand all variables in "/tmp/.boto"
+  envsubst < /tmp/.boto > /root/.boto
+else
+  echo 'Can not authorizaed. You should set service account authorization or HMAC authorization.'
```

Comment:
`Error: set the variables for GCS authentication.`
And, add `1>&2` since this message should be output to stderr.